apps/docs/docs/reference/ui/atoms/Link.mdx

```diff
@@ -40,8 +63,123 @@ For example, if someone wants to use a wrapper for any external link component (
 
 Besides those attributes, the following props are also supported:
 
-<PropsSection name="StorybookLink" />
+<PropsSection name="Link" />
 
-## Customization
+## Design Tokens
+
+<TokenTable>
+  <TokenRow token="--fs-link-min-width" value="auto" />
+  <TokenRow token="--fs-link-min-height" value="var(--fs-link-min-width)" />
+  <TokenRow
+    token="--fs-link-padding"
+    value="var(--fs-spacing-2) var(--fs-spacing-0)"
+  />
+  <TokenRow token="--fs-link-border-radius" value="var(--fs-border-radius)" />
+  <TokenDivider />
+  <TokenRow token="--fs-link-text-line-height" value="1.5" />
+  <TokenRow token="--fs-link-text-color" value="var(--fs-color-link)" isColor />
+  <TokenRow
+    token="--fs-link-text-color-visited"
+    value="var(--fs-color-link-visited)"
+    isColor
+  />
+  <TokenRow token="--fs-link-text-decoration" value="none" />
+  <TokenRow token="--fs-link-text-decoration-hover" value="underline" />
+  <TokenDivider />
+  <TokenRow
+    token="--fs-link-transition-function"
+    value="var(--fs-transition-function)"
+  />
+  <TokenRow
+    token="--fs-link-transition-property"
+    value="var(--fs-transition-property)"
+  />
+  <TokenRow
+    token="--fs-link-transition-timing"
+    value="var(--fs-transition-timing)"
+  />
+</TokenTable>
+
+---
+
+## Variants
+
+### Inverse
+
+```tsx live
+<Link inverse={true}>Inverse Link</Link>
+```
+
+<TokenTable>
+  <TokenRow
+    token="--fs-link-inverse-text-color"
+    value="var(--fs-color-link-inverse)"
+    isColor
+  />
+  <TokenRow
+    token="--fs-link-inverse-text-color-visited"
+    value="var(--fs-link-inverse-text-color)"
+    globalValue="var(--fs-color-link-inverse)"
+    isColor
+  />
+</TokenTable>
+
+### Display
+
+```tsx live
+<Link variant="display">Display Link</Link>
+```
+
+<TokenTable>
+  <TokenRow
+    token="--fs-link-display-text-line-height"
+    value="var(--fs-link-text-line-height)"
+  />
+  <TokenRow
+    token="--fs-link-display-text-color"
+    value="var(--fs-color-text-display)"
+    isColor
+  />
+  <TokenRow
+    token="--fs-link-display-text-color-visited"
+    value="var(--fs-link-display-text-color)"
+    globalValue="var(--fs-color-text-display)"
+    isColor
+  />
+</TokenTable>
+
+### Inline
+
+```tsx live
+<Link variant="inine">Inline Link</Link>
+```
```

Comment:
For some reason, this link looks pink here
<img width="876" alt="image" src="https://user-images.githubusercontent.com/11613011/205942370-2fdbde48-d443-4dd4-8122-7a8973f99f20.png">

Reply:
Strange, im seeing...